src/components/form/Checkbox.vue

```diff
@@ -1,31 +1,33 @@
 <script>
-import { _EDIT } from '@/config/query-params';
```

Comment:
Please put this in a separate commit (I assume this is just straight copied from dashboard).

Reply:
done in commit 79fe043